tcms/xmlrpc/api/forms/testplan.py

```diff
@@ -0,0 +1,35 @@
+from django.forms import BooleanField, CharField, ModelChoiceField
+
+from tcms.management.models import Product, Version
+from tcms.testplans.forms import NewPlanForm
```

Comment:
`from tcms.testplans import forms as testplans_forms` and use it later for the base class. If one of the definitions is rearranged the class named `NewPlanForm` and defined in this module may conflict with the class using the same name and coming from the import! This can lead to all sort of hard to figure out bugs.